.travis.yml

```diff
@@ -16,6 +16,7 @@ android:
 
 before_install:
   - pip install --user codecov
+  - brew install infer
```

Comment:
Yup, most of CIs run on Linux including Travis and Circle (by default),
they have Mac OS machines but paid.

On Sat, 21 May 2016, 23:34 Niklas Baudy, <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/artem-zinnatullin/qualitymatters/pull/178#discussion_r64139900>
> :
>
> > @@ -16,6 +16,7 @@ android:
> >
> >  before_install:
> >    - pip install --user codecov
> > +  - brew install infer
>
> Oh CircleCI is running on linux? Anyways so far it does not matter since
> the gradle plugin fails and this one needs to be fixed first.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/artem-zinnatullin/qualitymatters/pull/178/files/ef8d3cef5d83faf41bb3ae505bf1947dbaf9175e#r64139900>
>